.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     },
     extends: [require.resolve('tsjs/eslint-config')],
     parserOptions: {
-        project: 'tsconfig.json',
+        project: ['./tsconfig.build.json', './tsconfig.test.json'],
```

Comment:
I think the project `project: './tsconfig.test.json'` should be in the `overrides` section instead, but I'm not 100% certain

Reply:
I run into an issue with the `parserOptions.projects`for the typescript-eslint/parser when I remove the `./tsconfig.test.json` from the project

I think it might be required there because it will make it possible for the build to exclude the spec files and the other to include specs as per @GermainBergeron